### DISCUSSION TITLE
how to user write_file to insert some commans in a same file.

### DISCUSSION THREAD

**Discussion by MicShao** [nornir-automation/nornir#653] [Q&A]
Dear David:
         how can I use write_file in nornir_utils.plugins.tasks.files to let outputs from different commands write in a same file.
The coding is :
from nornir import InitNornir
from nornir_netmiko import netmiko_send_command
from nornir_utils.plugins.functions import print_result
from nornir_utils.plugins.tasks.files import write_file
from datetime import date
import time
start_time = time.time()

def analysis_results(tasks):
    for cmd in commands:
        r = task.run(task=netmiko_send_command, command_string=cmd)
        task.run(task=write_file, content=r.result, filename=str(task.host.name)+'-'+str(date.today())+'-analysis.txt')

Regards
Michael

**Comment by dbarrosop** [comment]
Please, format your issue properly so it's easy to read. Thanks

**Reply by MicShao** [reply]
I want to use write_file to write the results of different commands in a same file.
And I just find a factor (append) of write_file can deal with this issue.
No worries.
Thank you!

**Reply by HaiderNL** [reply]
Have you got it solved ?
I'm trying the same thing with luck.
The goal is simply saving the output of a show command(s) to a file instead of printing it with print_result utility.
https://nornir.tech/nornir_utils/html/tutorials/write_file.html#Example
